Modules/Microsoft365DSC/Examples/Resources/AADAuthenticationMethodPolicy/1-AADAuthenticationMethodPolicy-Example.ps1

```diff
@@ -0,0 +1,43 @@
+<#
+This example is used to test new resources and showcase the usage of new resources being worked on.
+It is not meant to use as a production baseline.
+#>
+
+Configuration Example
+{
+    param(
+        [Parameter()]
+        [PSCredential]
+        $Credscredential
```

Comment:
Again, since we are not using the CredsCredential parameter in the example, please have it removed from all examples.